core/tests/snapshots.rs

```diff
@@ -218,7 +218,7 @@ fn run_bank_forks_snapshot_n<F>(
         accounts_package_sender,
     };
     for slot in 1..=last_slot {
-        let mut bank = Bank::new_from_parent(&bank_forks[slot - 1], &Pubkey::default(), slot);
+        let mut bank = Bank::new_from_parent(bank_forks.get(slot - 1).unwrap(), &Pubkey::default(), slot);
```

Comment:
Replacing `[i]` with `.get(i).unwrap()` is just more verbose, no?
If it is replaced with `.get(i).expect("... reasons ...")` then you are providing more information.  Which could be useful.
But what is the advantage if it is just `.unwrap()`?

This change is repeated a few time.

Reply:
Yeah, agreed; I can't remember why I changed this to start with but I flipped this specific one back. I'll need to backburner this for Wed, but will run through and to find + revert these changes made elsewhere in the PR

Reply:
Ok, I have reverted the
```
bank_forks.get(x).unwrap()
```
instances that I changed back to
```
bank_forks[x].clone()
```
There are a bunch of instances where test code still does `bank_forks.get(x).unwrap()`; however, I think changing all of those is outside of the scope of this PR and would make this PR too noisy so I'm going to leave those as-is.